roadmap.md

```diff
@@ -50,51 +43,26 @@ state.
 
    Project board: https://github.com/orgs/kubernetes/projects/137
 
-1. **Signing of release artifacts (Secure)**
-
-   Outcome: Being able to ship signed release artifacts, which includes
-   container images in the first iteration as well as all artifacts following
-   on.
-
-   Enhancement: https://github.com/kubernetes/enhancements/issues/3031
-
-1. **Migrate away from GCS bucket: gs://kubernetes-release**
-
-   Outcome: Ensure we migrate away from gs://kubernetes-release, use a community-owned Object Storage bucket for binaries releases, serve the binaries published through a CDN service.
-
-### To be done (TODO)
-
-1. **Enhance Kubernetes binary artifact management (Consumable)**
-
-   https://github.com/kubernetes/sig-release/issues/1372
-
-   Enhancement: _none_
-
-   Outcome: Being able to promote files as artifacts and using this mechanism
-   for Kubernetes releases.
-
-1. **Define and collect metrics about Kubernetes releases (Introspectable)**
+1. **Make Kubernetes releases more flexible**
 
-   https://github.com/kubernetes/sig-release/issues/1527
+   Outcome: Ensure we migrate away from gs://kubernetes-release, use a
+   community-owned Object Storage bucket for binaries releases, serve the
+   binaries published through a CDN service.
```

Comment:
@ameukam is it fine if we make this a bit less specific? This way we could integrate topics like:

- Allow creating kubernetes releases from (private) forks
- Add parameters to krel to support other destinations (buckets)

Reply:
That will increase the timeframe to implement this. 
We may end up in a situation where the `gs://kubernetes-release` bucket is unavailable for the release process.
Better to do this ASAP (in 1.30).

Reply:
Good, reverted back to the original state.